caom2-artifact-resolvers/src/main/java/ca/nrc/cadc/caom2/artifact/resolvers/MastResolver.java

```diff
@@ -40,8 +41,11 @@
  * @author jeevesh
  */
 public class MastResolver implements StorageResolver {
+
     public static final String SCHEME = "mast";
-    private static final String MAST_BASE_ARTIFACT_URL = "https://masttest.stsci.edu/partners/download/file/";
+    private static final Logger log = Logger.getLogger(MastResolver.class);
+    private static final String MAST_BASE_ARTIFACT_URL = "http://mastpartners.stsci.edu/portal/Download/file";
+    private static final String CANNOT_GET_URL = "Can't generate URL from URI.";
```

Comment:
This message is not used anywhere and can be removed.

Also, I noticed that the header comment is not the correct open-source one... probably slipped by on the first review